material-ui-chip-input/build.boot

```diff
@@ -2,7 +2,7 @@
   :resource-paths #{"resources"}
   :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
```

Comment:
Hmm, I forgot that material-ui comes with its own React? If that is still the case, `cljsjs/react` should be dependency here.